src/app/welcome/_components/Header.tsx

```diff
@@ -0,0 +1,194 @@
+
+const navigationItems = [
+  {
+    href: "#about-us",
+    label: "デジクリとは？",
+    icon: MaterialSymbolsInfo,
+    type: "toc" as const,
+  },
+  {
+    href: "#teams",
+    label: "班紹介",
+    icon: MaterialSymbolsGroups,
+    type: "toc" as const,
+  },
+  {
+    href: "#faq",
+    label: "よくある質問",
+    icon: MaterialSymbolsHelp,
+    type: "toc" as const,
+  },
+  {
+    href: "https://forms.gle/cY25Kc6fssqv2tZz9",
+    label: "入部受付",
+    icon: MaterialSymbolsOpenInNew,
+    type: "external" as const,
+    isJoinUs: true,
+  },
+];
+
+const socialLinks = [
+  {
+    href: "https://x.com/sitdigicre",
+    label: "X (Twitter)",
+    icon: SimpleIconsX,
+  },
+  {
+    href: "https://www.youtube.com/@sitdigicre",
+    label: "YouTube",
+    icon: SimpleIconsYoutube,
+  },
+  // {
+  //   href: "https://line.me/ti/g2/UfhEOyteJRjyQLm-c9Mmhj_9wvIKCBDEohh_WQ",
+  //   label: "LINE オープンチャット",
+  //   icon: SimpleIconsLine,
+  // },
+  {
+    href: "https://digicre.net",
+    label: "ウェブサイト",
+    icon: MaterialSymbolsLanguage,
+  },
+  {
+    href: "mailto:contact@digicre.net",
+    label: "お問い合わせ",
+    icon: MaterialSymbolsMail,
+  },
+];
+
+export default function Header() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const handleLinkClick = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <header className="relative">
+      <input
+        type="checkbox"
+        name="toggle"
+        id="header-toggle"
+        className="hidden"
+        checked={isOpen}
+        onChange={handleToggle}
+      />
+
+      <label
+        htmlFor="header-toggle"
+        className={`fixed top-0 left-0 z-10 cursor-pointer rounded-br-[16px] bg-[#00b0f0] p-4 shadow-[0_2px_8px_0_rgba(0,0,0,0.5)] transition-colors duration-300 ease-in-out hover:bg-[#71d9ff] lg:hidden ${!isOpen ? "block" : "hidden"}`}
+        aria-label="メニューを開く"
+      >
+        <MaterialSymbolsMenu className="block h-8 w-8 text-white" />
+      </label>
+
+      <div
+        className={`fixed top-0 left-0 z-10 flex h-full w-full flex-col gap-8 overflow-y-auto bg-[#202020] px-4 py-16 text-white lg:w-1/4 ${isOpen ? "block" : "hidden lg:flex"}`}
+      >
+        <label
+          htmlFor="header-toggle"
+          className="absolute top-0 left-0 z-10 cursor-pointer rounded-br-[16px] bg-[#00b0f0] p-4 shadow-[0_2px_8px_0_rgba(0,0,0,0.5)] transition-colors duration-300 ease-in-out hover:bg-[#71d9ff] lg:hidden"
+          aria-label="メニューを閉じる"
+        >
+          <MaterialSymbolsClose className="block h-8 w-8 text-white" />
+        </label>
+
+        <Link
+          href="/welcome"
+          className="mx-auto block h-[48px] w-[176px] text-white"
+          aria-label="デジクリ"
+          onClick={handleLinkClick}
+        >
+          <DigicreLogo className="h-12 w-[176px] text-white" />
+        </Link>
+
+        <nav className="overflow-hidden rounded-2xl bg-[#404040]">
+          <ul>
```

Comment:
`<nav>` にメニューの CSS 諸々が適用されているため，ウィンドウの高さが小さいとメニュー項目が途切れてしまいます。ブラウザーの開発者ツールで軽く検証した限り，`<ul>` 側で適用すると従前と変わらない動作を実現可能です。

<img width="2240" height="840" alt="Image" src="https://github.com/user-attachments/assets/70fdb8c3-3b4c-4151-93e1-7c2b54209eae" />

```suggestion
        <nav>
          <ul className="overflow-hidden rounded-2xl bg-[#404040]">
```